build.gradle

```diff
@@ -1,19 +1,21 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+
   ext {
-    kotlin_version = '1.2.51'
+    kotlin_version = '1.3.11'
     compileSdkVersion = 28
     targetSdkVersion = 28
     minSdkVersion = 19
-    support_lib_version = '27.1.1'
   }
+
   repositories {
     google()
     jcenter()
   }
+
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
+    classpath 'com.android.tools.build:gradle:3.3.0-rc02'
```

Comment:
Might be a little early for this change. I think it's better to leave this at 3.2.1 for now, since any tutorial writer using 3.3.0-rc02 and up will be prompted to change it anyway when they open the template project.

Reply:
Got it. Reverting this change.